api_views/books.py

```
@@ -77,4 +84,28 @@ def get_by_title(book_title):
                 }
                 return Response(json.dumps(responseObject), 200, mimetype="application/json")
             else:
-                return Response(error_message_helper("Book not found!"), 404, mimetype="application/json")
\ No newline at end of file
+                return Response(error_message_helper("Book not found!"), 404, mimetype="application/json")
+
+
+def delete_book(book_id):
+    req_data = request.json()
```

Error handling in delete_book() function uses bare except clause which is too broad. Specify the expected jsonschema.ValidationError exception instead.